### PR TITLE
Application dashboard doc

### PR DESCRIPTION
**Pull request by whitetiger1399**
Signed-off-by: Ritik Srivastava <ritik@chaosnative.com>

**What this PR does / why we need it**:
Documentation for the Application dashboard as part of Chaos Observibilty added

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/user-guides/editing-queries-app-dashboard.md [hunk 1]
```diff
@@ -4,4 +4,76 @@ title: Editing Queries for Panels in Application Dashboard
...
+<figure>
+<img src={require('../assets/user-guides/observability/setup/create-dashboard-edit-query.png').default} />
+<i>Editing a query</i>
+</figure>
+
+- **Copy query:** Click to copy the Promql query.
+- **Delete a query:** Click to delete the specific query.
+- **Hide query:**  Click to hide the query from the panel. This feature assists you in creating and tuning the graph without deleting the queries. 
+- **Queries:** you can edit the queries to get the appropriate visualisations of your application’s metrics. The number of queries present in the particular panel is also mentioned besides the `Queries` text. Please note that the metric, key and values fields provide you assistance in the generation of a Promql query. You may generate a Promql query with their assistance or if you already have a Promql query then you can directly add it in the query field.
+  - Metric: All the metrics which are valid as per the connected data source are listed in the dropdown. Select any metric as per your requirement. Also if you want to input a metric that is not listed in the dropdown then you may simply enter the metic name as the option of free-solo has been enabled.
```
Comment (Saranya-jena) [inline website/docs/user-guides/editing-queries-app-dashboard.md]: metic -> metric

File: website/docs/user-guides/editing-queries-app-dashboard.md [hunk 2]
```diff
@@ -4,4 +4,76 @@ title: Editing Queries for Panels in Application Dashboard
...
+  - Metric: All the metrics which are valid as per the connected data source are listed in the dropdown. Select any metric as per your requirement. Also if you want to input a metric that is not listed in the dropdown then you may simply enter the metic name as the option of free-solo has been enabled. 
+  - (Optional) Key & Values:  Based upon your selection of the metric, all the valid keys are listed in the dropdown. For a given key all the possible valid values are listed in the drop-down list in the Values field. You may select multiple values for the given key.
+  
+    To select and configure multiple keys, you can switch between all the keys available and select the corresponding values for them. All of your selections for the keys and their values are reflected in the Promql query generated based upon your selection.
+
+  - Query: This field can be auto-generated based upon your selections for the metric, key and values field or you can directly provide it with a Promqul query. The query editing option also comes along with intelligence for user assistance.
```
Comment (Saranya-jena) [inline website/docs/user-guides/editing-queries-app-dashboard.md]: Promqul -> Promql

File: website/docs/user-guides/manage-app-dashboard.md [hunk 3]
```diff
@@ -4,4 +4,86 @@ title: Creating and Updating an Application Dashboard
...
+
+- **Choose a dashboard-type:**
+<figure>
+<img src={require('../assets/user-guides/observability/setup/manage-application-dashboard-dashboard-type.png').default} />
+<i>Select dashboard type</i>
+</figure>
+
+You need to select the dashboard type from the following options:
+
+  - *Predefined dashboards:* Litmus provides a set of predefined dashboards for different applications and use-cases like Node metric, Pod metric, Sock Shop etc along with the chaos events and chaos exporter metrics.
```
Comment (vanshBhatia-A4k9) [inline website/docs/user-guides/manage-app-dashboard.md:57]:
```suggestion
  - *Predefined dashboards:* ChaosCenter provides a set of predefined dashboards for different applications and use-cases like Node metric, Pod metric, Sock Shop etc along with the chaos events and chaos exporter metrics.
```

Comment (whitetiger1399) [reply website/docs/user-guides/manage-app-dashboard.md:57]: For this use case we will be using Litmus

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]
Can you add Learn more and Before you Begin sections

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**